tokamak/1-no-currents/BOUT.inp

```diff
@@ -0,0 +1,309 @@
+# derivative methods
+
+[ddx]
+
+first = C2
+second = C2
+upwind = W3
+
+[ddy]
+
+first = C2
+second = C2
+upwind = W3
+
+[ddz]
+
+first = C2
+second = C2
+upwind = W3
+
+###################################################
+# Time-integration solver
+
+[solver]
+mxstep = 1000000
+
+# Note: If evolving neutrals, need preconditioning
+type = cvode
+use_precon = true
+
+maxl = 10
+
+ATOL = 1.0e-10  # absolute tolerance
+RTOL = 1.0e-5   # relative tolerance
+mxstep = 1000000  # Maximum internal steps per output
+
+[laplace]  # This is used for Delp2 operator
+all_terms = true
+nonuniform=true
+
+##################################################
+# Electrostatic potential solver
+
+[phiSolver]
+inner_boundary_flags = 0
+outer_boundary_flags = 16  # INVERT_SET  (2.8 * Te)
+
+all_terms = true
+nonuniform=true   # NOTE: Necessary to avoid numerical instability
+
+[laplacexy]  # 2D solver in X-Y
+pctype = hypre    # Preconditioner
+
+atol = 1e-12
+rtol = 1e-8
+
+core_bndry_dirichlet = false
+pf_bndry_dirichlet = true
+y_bndry_dirichlet = false
+
+include_y_derivs = true
+
+[aparSolver]
+inner_boundary_flags = 0
+outer_boundary_flags = 0
+
+all_terms = true
+nonuniform=true
+
+# general settings for the model
+
+[Hermes]
+
+#######################
+# Output variables
+output_ddt = false # Output time derivatives
+verbose = true
+
+#######################
+# Numerical dissipation
+numdiff = -0.01    # Numerical diffusion
+hyper = -1      # Hyper-diffusion
+ExBdiff = -1e-3  # Dissipation in ExB advection terms
+y_hyper_viscos = -1
+z_hyper_viscos = -0.01
+ne_hyper_z = -0.01
+pe_hyper_z = -0.01
+
+vepsi_dissipation = true  # Parallel dissipation on Ve-Vi
+vort_dissipation = false
+
+radial_buffers = true
+radial_inner_width = 4
+radial_outer_width = -1
+radial_buffer_D = 1
+
+# Flux limiters
+kappa_limit_alpha = 0.2 # SOLPS style heat flux limiter
+eta_limit_alpha   = 0.5 # SOLPS style viscosity limiter
+
+#######################
+# Electric field and Ohm's law
+electromagnetic = true  # Electromagnetic? Otherwise electrostatic
+FiniteElMass    = true  # Finite electron mass?
+
+# Electrostatic potential
+split_n0 = false  # Solve n=0 separately?
+split_n0_psi = false
+
+# NOTE: all currents switched off for fluid run
+j_diamag = false
+j_par = false
+
+pe_par         = true  # Parallel pressure gradient: Pe <-> Psi
+resistivity    = true  # Resistivity: Psi -> Pe
+thermal_flux   = true
+thermal_force  = true
+electron_viscosity = false
+ion_viscosity = false
+ion_viscosity_par = true
+thermal_conduction = true
+poloidal_flows = false   # Include ExB flows in the X-Y plane?
+ion_velocity = true      # Evolve ion parallel velocity?
+boussinesq = true   # Use Boussinesq approximation
+
+frecycle = 0.99
+
+carbon_fraction = 0.01
+
+excitation = true # Hydrogen neutral excitation radiation
+
+## Settings for 2D parallel closures
+sinks = false
+sink_invlpar =  0.2  # 5m parallel connection length
+sheath_closure = false
+drift_wave = false
```

Comment:
Not needed here? "2D" in the comment could be confusing when this input file is set up for 2D (axisymmetric) simulations...